bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -408,6 +413,7 @@ private Metadata createMetadata(String tableName) {
   private ResultScanner<Row> streamRows(ReadRowsRequest request) {
     expandPoolIfNecessary(this.bigtableOptions.getChannelCount());
 
+    Timer.Context timeContext = readRowsAsync.getRpcMetrics().timeRpc();
```

Comment:
nit: I think we've been inconsistent what to call local variables of this type but I think `timerContext` makes the most sense.